README.md

```diff
@@ -20,19 +20,35 @@ https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe-support-script
 
 ### Install the `healthcheck_lite` module
 
-To install the `healthcheck_lite` module, execute the following command on the Primary Master.
+#### Using Bolt
+
+Bolt is the preffered installation and execution method and can be used either on the master or another machine with connectivity to the master. Firstly, follow the instruction for [installing Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). PE customers are *strongly* encouraged to follow the note about directly installing the package if installing on a PE node.
+
+The following block of code will create a Boltdir under the logged in user's home directory and set up the module:
 
 ```bash
-puppet module install puppetlabs-healthcheck_lite --modulepath=/opt/puppetlabs/puppet/modules
+mkdir -p ~/Boltdir
+cd !$
+
+cat >Puppetfile <<EOF
```

Comment:
This will overwrite an existing `Puppetfile`
If so, should the directory be `Boltdir` or something unique to HCL?

Reply:
Changing it to an append it a good idea.  It may create a duplicate module error, but the error message is very clear.

```
root@pe-201901-master:~/Boltdir$ bolt puppetfile install
Puppetfiles cannot contain duplicate module names. Remove the duplicates of the following modules: stdlib
Failed to sync modules from /root/Boltdir/Puppetfile to /root/Boltdir/modules
```